File: maven-plugin/src/main/java/MavenUtils.java
import org.apache.maven.project.MavenProject;

/**
 * Utility methods for common Maven project operations.
 */
public class MavenUtils {
    
    
    /**
     * Format Maven project as "groupId:artifactId" key
     */
    public static String formatProjectKey(MavenProject project) {
        return project.getGroupId() + ":" + project.getArtifactId();
    }
    
    
}

File: maven-plugin/src/main/kotlin/MavenUtils.kt
import org.apache.maven.project.MavenProject

/**
 * Utility methods for common Maven project operations.
 */
object MavenUtils {
    
    /**
     * Format Maven project as "groupId:artifactId" key
     */
    @JvmStatic
    fun formatProjectKey(project: MavenProject): String {
        return "${project.groupId}:${project.artifactId}"
    }
}